grid/grid_network.py

```diff
@@ -39,7 +40,65 @@ def search(self, *query):
             tensor_set.append(worker.search(*query))
         return tensor_set
 
-    def serve_model(self, model, model_id):
+    def serve_model(self, model, model_id=None, encrypted=False):
```

Comment:
can we have to methods? serve_model and serve_encrypted_model instead of using an argument? This will be useful for having the same API between the grid_nodes and the grid_gateway.